package.json

```diff
@@ -4,18 +4,17 @@
   "description": "Lunie is the staking and governance platform for proof-of-stake blockchains.",
   "author": "Lunie International Software Systems Inc. <hello@lunie.io>",
   "scripts": {
-    "serve": "kill-port 9080 && vue-cli-service serve --port 9080",
-    "serve:dist": "live-server --port=9080 --no-browser ./dist",
-    "serve:mobile": "cross-env MOBILE_APP=true npm run serve",
     "build": "vue-cli-service build",
     "build:mobile": "cross-env MOBILE_APP=true npm run build",
+    "changelog": "simsala log --pending-path ./changes",
     "lint": "vue-cli-service lint",
     "lint:style": "vue-cli-service lint:style",
-    "changelog": "simsala log --pending-path ./changes",
     "release": "git checkout develop & git pull & git push origin develop:release",
     "release:android": "git checkout develop && node ./tasks/bump-android-version.js && git commit android/app/build.gradle -m 'updated android app version' && git push origin develop:release-android",
     "release:ios": "git checkout develop && node ./tasks/bump-ios-version.js && git commit ios/App/App/Info.plist -m 'updated ios app version' && git push origin develop:release-ios",
-    "serve:ios": "npx cap open ios & npm run serve",
+    "serve": "kill-port 9080 && vue-cli-service serve --port 9080",
```

Comment:
why?^^

Reply:
this is just in alphabetical order now

i removed `serve:ios` because it isn't necessary

serve works if the correct url is in the capacitor config (it wasn't - it got removed somehow)

Reply:
really not needed change which can cause conflicts. ;)